Formula/gnutls.rb

```diff
@@ -77,6 +77,10 @@ def post_install
 
     pkgetc.mkpath
     (pkgetc/"cert.pem").atomic_write(valid_certs.join("\n"))
+
+    # Touch gnutls.go to avoid Guile recompilation.
+    # See https://github.com/Homebrew/homebrew-core/pull/60307#discussion_r478917491
+    touch "#{lib}/guile/3.0/site-ccache/gnutls.go"
```

Comment:
If you're gonna make the guile dependency optional I think this needs to check if guile is installed before creating the file.

Reply:
But this is part of the bindings that get installed in the gnutls cellar, no? So it will always exist since we always build it (even they can't be used unless you have Guile installed). If we install Guile later then we would not know we have to touch that file.

Reply:
Seems fine to me.